test/seqan3-test.cmake

```diff
@@ -55,6 +55,9 @@ set (SEQAN3_TEST_CLONE_DIR "${PROJECT_BINARY_DIR}/vendor/googletest")
 # ----------------------------------------------------------------------------
 # Interface targets for the different test modules in seqan3.
 # ----------------------------------------------------------------------------
+if (WIN32)
+    set(AHLWAPI shlwapi.lib)
+endif()
```

Comment:
Here [MS docs.](https://docs.microsoft.com/en-us/windows/desktop/api/shlwapi/)
They documented here [google/benchmark: Known Issues- Windows with CMake](https://github.com/google/benchmark/blob/847c0069021ade355b7678a305f3ac4e4d6f7e79/README.md#windows-with-cmake) and [Microsoft](https://docs.microsoft.com/en-us/windows/desktop/api/shlwapi/nf-shlwapi-pathfileexistsa) 
it fixes linker bug(?) in gbenchmark: `unresolved external symbol __imp_SHGetValueA ?? ` I persistently found: https://github.com/qPCR4vir/seqan3/commit/a1b603860c96905786e0e69c06938bc31bc43c71
People said: [The dependency on shlwapi.lib gets injected through a .obj that is packaged inside benchmark.lib (compiled from here), so it needs to be present on the linker command line for anyone who links against benchmark.lib.](https://github.com/google/benchmark/issues/202#issuecomment-243373593) and [ It is, because Shlwapi.lib library does not get linked automatically, ever. The headers are not in Windows.h either, the header is pulled in by ../src/sysinfo.cc, which hides the fact that the linking is actually required, you won't be any the wiser until such time you are going to link to benchmark.lib.](https://github.com/google/benchmark/issues/634#issuecomment-404170260)

[examples CMakeLists.txt](https://github.com/google/benchmark/blob/6e259170011cd2fbac069b9d4ff83b42445a96cc/src/CMakeLists.txt#L36) and [here](https://github.com/apache/arrow/commit/cb9c52e4b1d81446ddcfaf4485a8284006dc68d0)

It was magically there. I saw a shlwapi.dll in c:\Windows\System32 and SysWOW64. That was not ich ;-). And it worked on all my PCs until now.